web/src/js/ducks/flows.js

```diff
@@ -117,6 +117,14 @@ export function update(flow, data) {
     return { type: REQUEST_ACTION }
 }
 
+export function update_content(flow, file) {
+    const body = new FormData()
+    body.append('file', file)
+    fetchApi(`/flows/${flow.id}/response/content`, {method: 'post',  body} )
```

Comment:
this fileupload should be used with a string or a file... i think it is better to create a file out of the string and then upload it as a file ? What do you think ?